dist/index.js

```diff
@@ -46,3 +51,26 @@ app.post('/webhooks', async (req, res) => {
 app.get('/', (req, res) => {
     res.status(200).send('Server running');
 });
+app.get('/search', async (req, res) => {
+    // const ans = await axios.get("https://shopping.google.com/search?q=nike")
+    const screen = {
+        width: 1920,
+        height: 1080
+    };
+    var option = new firefox_1.Options();
+    option.setBinary("C:/Program Files/Mozilla Firefox/firefox.exe");
```

Comment:
Are you sure this will work on remote deployment like heroku.

Reply:
chalega veereh